test/NuGet.Core.Tests/NuGet.ProjectModel.Test/RestoreAuditPropertiesTests.cs

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#nullable enable
+
+using Xunit;
+
+namespace NuGet.ProjectModel.Test
+{
+    public class RestoreAuditPropertiesTests
+    {
+        [Fact]
+        public void Equals_WithNull_ReturnsFalse()
+        {
+            // Arrange
+            var properties = new RestoreAuditProperties();
+
+            // Act & Assert
+            Assert.False(properties.Equals((RestoreAuditProperties?)null));
+            Assert.False(properties.Equals((object?)null));
+            Assert.False(properties == null);
+            Assert.False(null == properties);
+            Assert.True(properties != null);
+            Assert.True(null != properties);
+        }
+
+        [Fact]
+        public void Euqals_WithSameInstance_ReturnsTrue()
```

Comment:
```suggestion
        public void Equals_WithSameInstance_ReturnsTrue()
```

Reply:
I swear I was staring at this for over an hour, over the course of a few days, until I finally saw what the actual difference was 👓